lib/location_service/location_service.ex

```diff
@@ -13,14 +13,18 @@ defmodule LocationService do
   @cache Application.compile_env!(:dotcom, :cache)
   @ttl :timer.hours(24)
 
+  @filter ~r/,\s(MA|NH|RI),\s/
```

Comment:
I'm so glad you included the commas so that we don't filter out places/streets which happen to have these characters! :D